src/test/scala/nl.knaw.dans.easy.deposit/docs/DatasetXmlSpec.scala

```diff
@@ -0,0 +1,296 @@
+        <dcterms:alternative xml:lang="nld">alternative title2</dcterms:alternative>
+        <dcx-dai:contributorDetails>
+          <dcx-dai:author>
+            <dcx-dai:titles xml:lang="nld">Dr.</dcx-dai:titles>
+            <dcx-dai:initials>O.</dcx-dai:initials>
+            <dcx-dai:insertions>van</dcx-dai:insertions>
+            <dcx-dai:surname>Belix</dcx-dai:surname>
+          </dcx-dai:author>
+        </dcx-dai:contributorDetails>
+        <dcterms:rightsHolder>rightsHolder1</dcterms:rightsHolder>
+        <dcterms:rightsHolder>Dr. A.S. van Terix</dcterms:rightsHolder>
+        <dcterms:publisher xml:lang="nld">pub1</dcterms:publisher>
+        <dcterms:publisher xml:lang="nld">pub2</dcterms:publisher>
+        <dc:source xml:lang="nld">source1</dc:source>
+        <dc:source xml:lang="nld">source2</dc:source>
+        <dcterms:dateCopyrighted>2018-03-18</dcterms:dateCopyrighted>
+        <dcterms:valid>2018-03-17</dcterms:valid>
+        <dcterms:modified>2018-02-02</dcterms:modified>
+        <dcterms:issued>Groundhog day</dcterms:issued>
+        <dcterms:dateSubmitted>2018-03-22</dcterms:dateSubmitted>
+        <dcterms:license>http://creativecommons.org/publicdomain/zero/1.0</dcterms:license>
+      </ddm:dcmiMetadata>
+    )
+  )
+
+  private def parseTestResource(file: String) = Try {
+    (File(getClass.getResource("/manual-test")) / file).contentAsString
+  }.flatMap(DatasetMetadata(_))
+}
+
+trait DdmBehavior {
+  this: TestSupportFixture =>
+
+  val emptyDDM: Elem
+
+  lazy val triedSchema: Try[Schema] = Try { // loading postponed until we actually start validating
+    val factory = SchemaFactory.newInstance(XMLConstants.W3C_XML_SCHEMA_NS_URI)
+    val schemas = "http://dublincore.org/schemas/xmls/qdc/2008/02/11/dcterms.xsd" ::
+      "https://easy.dans.knaw.nl/schemas/md/ddm/ddm.xsd" :: Nil
+    factory.newSchema(schemas.map(xsd => new StreamSource(xsd)).toArray[Source])
+  }
+
+  // pretty provides friendlier trouble shooting for complex XML's than Utility.trim
+  private val prettyPrinter: PrettyPrinter = new scala.xml.PrettyPrinter(1024, 2)
+
+  /**
+   * @param input          to be converted to DDM
+   * @param subset         subset of generated DDM that should match expectedOutput
+   * @param expectedOutput members of <ddm:DDM> that should equal
+   *                       the members of subset(DatsetXm(input.get))
+   */
+  def validDatasetMetadata(input: Try[DatasetMetadata],
+                           subset: Elem => Elem = identity,
+                           expectedOutput: Seq[Node] = Seq.empty
+                          ): Unit = {
+    lazy val datasetMetadata = input
+      .doIfFailure{ case e => println(s"$e")}
+      .getOrElse(fail("test input should be a success"))
+    lazy val ddm = DatasetXml(datasetMetadata)
+      .doIfFailure{ case e => println(s"$e")}
+      .getOrElse(fail("can't create DDM from test input"))
+
+    it should "generate expected DDM" in {
+      if (expectedOutput.nonEmpty)
```

Comment:
@rvanheest 
Not completely satisfied here. Tried `assume` too, but that gives a worse wrong impression. But now it suggests the output is as expected while it is not asserted at all.